ariadne/enums.py

```diff
@@ -49,3 +107,132 @@ def set_default_enum_values(graphql_type: GraphQLEnumType):
     for key in graphql_type.values:
         if graphql_type.values[key].value is None:
             graphql_type.values[key].value = key
+
+
+def find_enum_values_in_schema(
+    schema: GraphQLSchema,
+) -> Generator[Union[ArgumentWithKeys, InputFieldWithKeys], None, None]:
+    object_types = (
```

Comment:
Not a biggest fan of iterating schema twice to create two lists that are then iterated over again.

Reply:
I have refactored it a little bit, now I rely on `functools`'s `singledispatch`.

Reply:
And now we iterating only once ofc ;d